### PR TITLE
Processunity - 68 - ABX Target to Impact Dashboard

### PR DESCRIPTION
**Pull request by raaayden**
Pushing new script for processunity on salesforce target account

https://processunity.wise-portal.com/request-summary.php?id=68

### CODE REVIEW

File: Clients/ProcessUnity/Analytics Script/processunity_target_account_log.sql [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+    account_leads._lead_created_date AS _cycle_date,
+    account_leads._lead_source,
+    '' AS _stage_name,
+    _lead_name,
+    _lead_email,
+    '' AS _opportunity_type
+  FROM account_leads
+  -- WHERE account_leads._lead_created_date IS NOT NULL
+  UNION ALL
+  SELECT DISTINCT
```
Comment (nazzatul21) [inline Clients/ProcessUnity/Analytics Script/processunity_target_account_log.sql:294]: This part not clear. Do leads and MQL is the same?

Comment (raaayden) [reply Clients/ProcessUnity/Analytics Script/processunity_target_account_log.sql:294]: This to differentiate the prospect status, which would show first as a Leads then may move to MQL if exist, same as opportunity. This is mainly to check on the account level, the analyst requested to have this kind of data.

File: Clients/ProcessUnity/Analytics Script/processunity_target_account_log.sql [hunk 2]
```diff
@@ -308,7 +308,7 @@ lead_status AS (
     _lead_email,
     '' AS _opportunity_type
   FROM account_leads
-  -- WHERE account_leads._WF_become_MQL_date IS NOT NULL
+  WHERE account_leads._WF_become_MQL_date IS NOT NULL
```
Comment (raaayden) [inline Clients/ProcessUnity/Analytics Script/processunity_target_account_log.sql:311]: @nazzatul21, I have made changes to enable this so the numbers of leads is different compared to the MQL as you previously mentioned. Thanks for the reminder !

Comment (nazzatul21) [reply Clients/ProcessUnity/Analytics Script/processunity_target_account_log.sql:311]: Okay nice! Now we can see the different

**Review by nazzatul21 — APPROVED** [review]

**Pull request merged**